tests/plots/test_summary_plot.py

```diff
@@ -34,3 +35,31 @@ def test_summary_plot(explainer):
     shap.plots.beeswarm(shap_values, show=False)
     plt.tight_layout()
     return fig
+
+
+@pytest.mark.parametrize(
+    "rng",
+    [
+        np.random.default_rng(167089660),
+        17,
+        np.random.SeedSequence(entropy=60767),
+    ],
```

Comment:
I've expanded the `rng` `type`s a bit to reflect the flexibility of SPEC 7, though there are even more possibilities than this.